### PR TITLE
Added GCP examples

### PR DESCRIPTION
**Pull request by sourabh2311**

### CODE REVIEW

**Comment by grastogi23** [comment]
@chaitanyaavi @Rohan-sss1

File: examples/gcp/cloudCreation/main.tf [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  zone    = var.zone
+  project = var.project_id
+}
+
+// Avi provider
+provider "avi" {
+  avi_username   = var.avi_username
+  avi_password   = var.avi_current_password
+  avi_controller = data.google_compute_instance.avi_controller.*.network_interface.0.network_ip[0]
+  avi_tenant     = "admin"
```
Comment (mayank-avinetworks) [inline examples/gcp/cloudCreation/main.tf:19]: can we add it in variables please?

File: examples/gcp/cloudCreation/main.tf [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    region_name           = var.region_name
+    se_project_id         = var.se_project_id
+    firewall_target_tags  = var.firewall_target_tags
+    zones                 = var.zones
+    cloud_credentials_ref = avi_cloudconnectoruser.gcp_user.id
+    # gcs_project_id        = "${var.gcs_project_id}"  // optional, read description for more info.
+    # gcs_bucket_name       = "${var.gcs_bucket_name}" // optional, read description for more info.
+  }
+
+  license_tier = "ENTERPRISE_18"
```
Comment (mayank-avinetworks) [inline examples/gcp/cloudCreation/main.tf:67]: please change this to ENTERPRISE

File: examples/gcp/clusterCreation/main.tf [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  name  = "${var.controller_name}-${count.index}"
+  zone  = var.zone
+}
+
+// Configure the Avi provider
+provider "avi" {
+  avi_username   = var.avi_username
+  avi_password   = var.avi_current_password
+  avi_controller = data.google_compute_instance.avi_controller.*.network_interface.0.network_ip[0]
+  avi_tenant     = "admin"
```
Comment (mayank-avinetworks) [inline examples/gcp/clusterCreation/main.tf:19]: please read this value form varable

**Comment by mayank-avinetworks** [comment]
@manojkumarjain FYI , please.

File: examples/gcp/cloudCreation/main.tf (hunk 1 above)
Comment (manojkumarjain) [reply examples/gcp/cloudCreation/main.tf:19]: Let us not hard code any variables

File: examples/gcp/clusterCreation/main.tf (hunk 3 above)
Comment (mayank-avinetworks) [reply examples/gcp/clusterCreation/main.tf:19]: @yograjshisode : this one is still left :)

**Pull request merged**